packages/theme/src/Components/Snackbar/index.tsx

```diff
@@ -224,3 +239,25 @@ export const CustomSnackbarProvider = memo<SnackbarProviderProps>((props) => {
         />
     )
 })
+
+export function useShowCostomSnackbar() {
+    const snackbar = useSnackbar()
+    const showSnackbar = useCallback(
+        (
+            message: string,
+            options: Partial<CustomSnackbarContentProps & { persist: boolean }> = {
+                variant: 'default',
```

Comment:
Perhaps in most scenarios, `success` is better as the default value

Reply:
It's all based on the design.